pkg/auth/providers/azure/clients/group_cache.go

```diff
@@ -0,0 +1,69 @@
+package clients
+
+import (
+	lru "github.com/hashicorp/golang-lru"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+	"time"
+
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+)
```

Comment:
Packages should be divided between standard and otherwise.

Reply:
Yup, missed in this file, I wish Goland would watch this and sort automatically. There is a way to do it, just needs tinkering with the settings.